scripts/bert/index.rst

```diff
@@ -37,3 +37,32 @@ Use the following command to fine-tune the BERT model for classification on the
    $ GLUE_DIR=glue_data python finetune_classifier.py --batch_size 32 --optimizer bertadam --epochs 3 --gpu --seed 1 --lr 2e-5
 
 It gets validation accuracy of 87.3%, whereas the the original Tensorflow implementation give evaluation results between 84% and 88%.
+
+BERT for SQuAD
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+GluonNLP provides the following example script to fine-tune SQuAD with pre-trained
+BERT model.
+
+Use the following command to fine-tune the BERT model for SQuAD1.1 dataset.
+
+Note that this will require more than 12G of GPU memory.
+ 
+.. code-block:: console
+
+    $ python finetune_squad.py --optimizer adam --batch_size 12 --lr 3e-5 --epochs 2 --gpu
+
+If you are using less than 12G of GPU memory, you can use the following command to achieve a similar effect. But need Mxnet>1.5.0
+
+Note that this will require approximately no more than 8G of GPU memory. If your GPU memory is too small, you can adjust **accumulate** and **batch_size**.
+
+.. code-block:: console
+
+    $ python finetune_squad.py --optimizer bertadam --accumulate 2 --batch_size 6 --lr 3e-5 --epochs 2 --gpu
```

Comment:
Why is bertadam used in this line and adam is used in line 53? Is that a typo?

Reply:
Grad accumulation is supposed to work irrespective of the choice of optimizer. What did you observe when adam is used with grad accumulation?

Reply:
I have not tested how adam works with gradient accumulation. But the above results are obtained with adam